client/src/pages/profile/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { StaffUpdateData } from '../../types';
+
+export const sanitizeData = (data: StaffUpdateData) => ({
+  password: data.password,
+});
```

Comment:
This is not a very safe sanitization strategy, we should just expose a new endpoint under the auth module called

```
auth/reset-password
```

That allows users to self update their passwords. The endpoint has a dto for just accepting a password field.